images/fcos-bastion-image/root/usr/bin/dnsmasq-init.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+PODMAN_DNSMASQ_BASEDIR=/var/opt/dnsmasq
+tftp_dir=${PODMAN_DNSMASQ_BASEDIR}/tftpboot
+password_hash=$(<"/var/opt/ignitions/password_hashes")
+
+# create ignition files
+pdm_opts="--rm --interactive --security-opt label=disable -v /var/opt/ignitions:/ign_files -v /opt/html:/output -w /ign_files"
+butane_img="quay.io/coreos/butane:release"
+
+podman run ${pdm_opts} ${butane_img} --pretty --strict --files-dir=./ wipe_disk.bu -o /output/wipe_disks.ign
+podman run ${pdm_opts} ${butane_img} --pretty --strict --files-dir=./ -o /output/shell.ign  <<< $(envsubst < shell.bu)
+
+# Extract grubaa64.efi and grubx64.efi from centso:stream9 RPMs
+mkdir -p ${PODMAN_DNSMASQ_BASEDIR}/tftpboot
+podman run --rm -t --privileged --name grubefi -v ${tftp_dir}:/tmp/output quay.io/centos/centos:stream9 <<EOF
+dnf install -y cpio
+cd /tmp
+curl -L -O $(dnf repoquery --location grub2-efi-aa64 --forcearch aarch64 | tail -n 1)
+curl -L -O $(dnf repoquery --location grub2-efi-x64 --forcearch x86_64 | tail -n 1)
+mkdir -p {aarch64,x86_64,output}
+
+cd /tmp/aarch64
+rpm2cpio /tmp/grub2-efi-aa64* | cpio -idmv
+mv boot/efi/EFI/centos/*.efi /tmp/output/
+
+cd /tmp/x86_64
+rpm2cpio /tmp/grub2-efi-x64* | cpio -idmv
+mv boot/efi/EFI/centos/*.efi /tmp/output/
+EOF
+
+# download latest Fedora CoreOS
```

Comment:
This should be another systemd unit, perhaps to run periodically.

Reply:
Done